Userland/Utilities/hexdump.cpp

```diff
@@ -12,11 +12,19 @@
 
 static constexpr size_t LINE_LENGTH_BYTES = 16;
 
+enum class STATES {
+    PRINT,
+    PRINT_FILLER,
+    NOPRINT
+};
```

Comment:
```suggestion
enum class State {
    Print,
    PrintFiller,
    DontPrint,
};
```

Please see the project coding style (https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md)
> Enum members should use InterCaps with an initial capital letter